rule-packs/aws-config.json

```diff
@@ -733,8 +733,8 @@
     "queries": [
       {
         "name": "query0",
-        "query": "FIND aws_vpc AS vpc THAT !LOGS AS flowLog * AS destination RETURN vpc.tag.AccountName, vpc.displayName, vpc.vpcId",
-        "version": "v1"
+        "query": "FIND aws_vpc AS vpc THAT !LOGS * with _type^='aws_' RETURN vpc.tag.AccountName, vpc.displayName, vpc.vpcId",
+        "version": "v2"
```

Comment:
I think this should still be version v1 because it refers to the version of the alerts engine being used, not the version of the query. Would need someone else to confirm though.

Reply:
Just updated to go back to v1